.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
```

Comment:
```suggestion
Resolves #ISSUE.
<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:

Partially addresses:
- ISSUE

Addresses sub-task (n) in:
- ISSUE
-->
```

Reply:
Also note that this section is optional, unlike the other sections in the PR ("High-level description" / "Overview" & Checklist). It's optional because there won't always be an issue associated with every PR.

Reply:
You need to issue another edit suggestion, as this one seems to have conflicted.

Reply:
_If_ this section is needed, that's fine, but seeing this makes me think there are other issues in the workflow.